ReleaseNotes-2.8.1

```diff
@@ -19,4 +19,4 @@ Changes:
 Bugfixes:
 =========
 
-* none
+* [webui] Errors handling of MissingParameterError exception in webui for live build log
```

Comment:
Could you rephrase this to a real sentence?:-)

Maybe "Fixes bug in live build log when no architecture or repository parameter was given" (or when it was invalid)